### ISSUE TITLE
Links not working anymore since 1.4.5 update (Windows 10)

### ISSUE THREAD

**Issue by ccrowles** [kotatogram/kotatogram-desktop#258]
### Steps to reproduce

Click a random link inside a chat.

### Expected behaviour

Link opens in the browser that's set as default (in my case it's Vivaldi Browser 4.1.2369.11 64bit stable).

### Actual behaviour

Nothing happens. 

**Additional notes:**

- In 1.4.4 I did not have this issue at all. 
- I updated to 1.4.5 via the Win64 installer posted on the [official telegram channel](https://t.me/kotatogram). Usually I check for updates directly from Kotatogram but this time I forgot and updated it from the exe, which I'm guessing it's why Kotatogram settings and login got reset (not a big problem), but not sure if that might have caused the "links not working" bug too.

### Operating system

Windows 10 Pro version 20H2

### Version of Kotatogram Desktop

1.4.5 (TD 3.1.1)

### Installation source

Static binary from official website

### Logs

```shell
[2021.12.05 00:35:50] Launched version: 3001001, install beta: [TRUE], dev: 0, debug mode: [FALSE]
[2021.12.05 00:35:50] Executable dir: C:/Users/Crowles/AppData/Roaming/Kotatogram Desktop/, name: Kotatogram.exe
[2021.12.05 00:35:50] Initial working dir: C:/Users/Crowles/AppData/Roaming/Kotatogram Desktop/
[2021.12.05 00:35:50] Working dir: C:/Users/Crowles/AppData/Roaming/Kotatogram Desktop/
[2021.12.05 00:35:50] Command line: C:\Users\Crowles\AppData\Roaming\Kotatogram Desktop\Kotatogram.exe
[2021.12.05 00:35:50] Executable path before check: C:/Users/Crowles/AppData/Roaming/Kotatogram Desktop/Kotatogram.exe
[2021.12.05 00:35:50] Logs started
[2021.12.05 00:35:50] Connecting local socket to Global\57bca493b5ee83ec5f7192b9b9545cef-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2021.12.05 00:35:50] This is the only instance of Telegram, starting server and app...
[2021.12.05 00:35:50] Moved logging from 'C:/Users/Crowles/AppData/Roaming/Kotatogram Desktop/log_start0.txt' to 'C:/Users/Crowles/AppData/Roaming/Kotatogram Desktop/log.txt'!
[2021.12.05 00:35:50] Primary screen DPI: 96
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.12.05 00:35:50] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2021.12.05 00:35:51] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.12.05 00:35:51] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2021.12.05 00:35:51] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.12.05 00:35:51] Using DirectX compiler 'C:\Windows\SYSTEM32\d3dcompiler_47.dll'.
[2021.12.05 00:35:51] App Info: reading settings...
[2021.12.05 00:35:51] App Info: reading encrypted settings...
[2021.12.05 00:35:51] Lang Info: Loaded cached, keys: 3695
[2021.12.05 00:35:51] OpenAL Logging Level: (not set)
[2021.12.05 00:35:51] Audio Playback Devices: OpenAL Soft on Altoparlanti (High Definition Audio Device);OpenAL Soft on VX228 (NVIDIA High Definition Audio);OpenAL Soft on Digital Audio (S/PDIF) (High Definition Audio Device)
[2021.12.05 00:35:51] Audio Playback Default Device: OpenAL Soft on Altoparlanti (High Definition Audio Device)
[2021.12.05 00:35:51] Audio Capture Devices: OpenAL Soft on Microfono (HD Pro Webcam C920)
[2021.12.05 00:35:51] Audio Capture Default Device: OpenAL Soft on Microfono (HD Pro Webcam C920)
[2021.12.05 00:35:51] App Info: reading accounts info...
[2021.12.05 00:35:51] App Info: reading encrypted info...
[2021.12.05 00:35:51] App Info: reading map...
[2021.12.05 00:35:51] App Info: reading encrypted map...
[2021.12.05 00:35:51] App Info: reading encrypted user settings...
[2021.12.05 00:35:51] App Info: encrypted user settings read.
[2021.12.05 00:35:51] App Info: reading encrypted mtp data...
[2021.12.05 00:35:51] MTP Info: read keys, current: 5, to destroy: 0
[2021.12.05 00:35:51] Map read time: 4
[2021.12.05 00:35:51] App Info: reading encrypted mtp config...
[2021.12.05 00:35:51] Export Info: Destroy top bar by controller removal.
[2021.12.05 00:35:51] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2021.12.05 00:35:51] OpenGL Profile: OpenGLES.
[2021.12.05 00:35:51] OpenGL Renderer: ANGLE (NVIDIA, NVIDIA GeForce GTX 1050 Direct3D11 vs_5_0 ps_5_0, D3D11-27.21.14.6663)
[2021.12.05 00:35:51] OpenGL Vendor: Google Inc. (NVIDIA)
[2021.12.05 00:35:51] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.7 git hash: 0bb011f9e403)
[2021.12.05 00:35:51] OpenGL Extensions: GL_EXT_robustness, GL_OES_compressed_EAC_R11_signed_texture, GL_ANGLE_explicit_context, GL_EXT_read_format_bgra, GL_ANGLE_explicit_context_gles1, GL_ANGLE_instanced_arrays, GL_CHROMIUM_bind_generates_resource, GL_OES_surfaceless_context, GL_OES_fbo_render_mipmap, GL_OES_mapbuffer, GL_EXT_occlusion_query_boolean, GL_NV_pixel_buffer_object, GL_EXT_discard_framebuffer, GL_OES_depth24, GL_ANGLE_framebuffer_blit, GL_EXT_color_buffer_float, GL_NV_fence, GL_OES_EGL_image_external_essl3, GL_CHROMIUM_copy_compressed_texture, GL_EXT_frag_depth, GL_OES_rgb8_rgba8, GL_NV_pack_subimage, GL_OES_vertex_array_object, GL_EXT_texture_compression_bptc, GL_EXT_texture_storage, GL_EXT_unpack_subimage, GL_OES_compressed_EAC_R11_unsigned_texture, GL_ANGLE_depth_texture, GL_OES_draw_elements_base_vertex, GL_ANGLE_get_serialized_context_string, GL_EXT_draw_buffers, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_ANGLE_texture_multisample, GL_OES_draw_buffers_indexed, GL_EXT_shader_texture_lod, GL_NV_EGL_stream_consumer_external, GL_OES_compressed_ETC2_RGBA8_texture, GL_ANGLE_multiview_multisample, GL_CHROMIUM_lose_context, GL_ANGLE_pack_reverse_row_order, GL_KHR_debug, GL_EXT_blend_minmax, GL_ANGLE_provoking_vertex, GL_ANGLE_translated_shader_source, GL_ANGLE_texture_compression_dxt5, GL_ANGLE_get_tex_level_parameter, GL_EXT_blend_func_extended, GL_EXT_draw_buffers_indexed, GL_ANGLE_texture_usage, GL_OES_EGL_image_external, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_ANGLE_lossy_etc_decode, GL_OES_texture_float_linear, GL_EXT_color_buffer_half_float, GL_EXT_texture_compression_s3tc_srgb, GL_OES_depth32, GL_OES_texture_half_float, GL_OES_texture_npot, GL_OES_EGL_image, GL_CHROMIUM_copy_texture, GL_OES_compressed_ETC2_sRGB8_texture, GL_EXT_EGL_image_external_wrap_modes, GL_EXT_multisampled_render_to_texture, GL_CHROMIUM_bind_uniform_location, GL_EXT_disjoint_timer_query, GL_WEBGL_video_texture, GL_ANGLE_memory_size, GL_OES_texture_border_clamp, GL_OES_texture_float, GL_ANGLE_robust_client_memory, GL_OES_element_index_uint, GL_OES_packed_depth_stencil, GL_ANGLE_request_extension, GL_EXT_debug_marker, GL_EXT_texture_compression_dxt1, GL_EXT_texture_rg, GL_ANGLE_framebuffer_multisample, GL_OES_compressed_EAC_RG11_signed_texture, GL_OES_compressed_ETC2_RGB8_texture, GL_OES_standard_derivatives, GL_EXT_sRGB, GL_ANGLE_copy_texture_3d, GL_OVR_multiview, GL_EXT_instanced_arrays, GL_KHR_parallel_shader_compile, , GL_ANGLE_multi_draw, GL_CHROMIUM_sync_query, GL_EXT_texture_compression_rgtc, GL_OES_texture_half_float_linear, GL_EXT_debug_label, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_ANGLE_client_arrays, GL_ANGLE_texture_compression_dxt3, GL_OES_get_program_binary, GL_OES_texture_stencil8, GL_OVR_multiview2, GL_EXT_texture_filter_anisotropic, GL_EXT_texture_format_BGRA8888, GL_ANGLE_program_cache_control, GL_EXT_float_blend, GL_ANGLE_base_vertex_base_instance, GL_EXT_map_buffer_range, GL_EXT_draw_elements_base_vertex, GL_EXT_texture_norm16, GL_OES_compressed_EAC_RG11_unsigned_texture
[2021.12.05 00:35:51] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_flexible_surface_compatibility, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2021.12.05 00:35:51] OpenGL: QOpenGLContext created, version: 3.3.
[2021.12.05 00:35:51] OpenGL: [TRUE] (OverlayWidget)
[2021.12.05 00:35:53] Message Info: bad message notification received (error_code 33) for msg_id = 7037995198406639580, seq_no = 56
[2021.12.05 00:35:53] Message Info: bad message notification received, msgId 7037995198406639580, error_code 33
[2021.12.05 00:35:53] API Warning: not loaded minimal channel applied.
```

**Comment by RadRussianRus** [comment]
Fixed in 490b29b31c2bc03244820d610a3494b4a4ec0bea, fix will arrive in next release.

**Comment by ccrowles** [comment]
Thanks!
